pandas/tests/frame/test_query_eval.py

```diff
@@ -1038,3 +1038,34 @@ def test_invalid_type_for_operator_raises(self, parser, engine, op):
 
         with pytest.raises(TypeError, match=msg):
             df.eval('a {0} b'.format(op), engine=engine, parser=parser)
+
+
+class TestDataFrameQueryBacktickQuoting(object):
+
+    def setup_method(self, method):
```

Comment:
we don't use setup_method / teardown idioms any longer, use pytest fixtures

Reply:
Using pytest fixtures now.